### PR TITLE
more commands

### LINKED ISSUE #2: relic lookup command port

### PR DESCRIPTION
**Pull request by aliasfalse**
added commands, removed some parameters on pricecheck and whereis

@nspacestd if you have time

i've also closed #2, #5, #7, #8, #9, #10, #15, #18, #19

### CODE REVIEW

File: src/commands/Syndicates.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        });
+
+        const color = syndicateInValues ? 0x00ff00 : 0xff0000;
+        const embed = {
+          color,
+          title: 'Worldstate - Syndicate Nodes',
+          url: 'https://warframe.com',
+          description: `Current Missions for Syndicate: ${type}`,
+          thumbnail: {
+            url: 'https://raw.githubusercontent.com/aliasfalse/genesis/master/src/resources/conclave.png',
```
Comment (nspacestd) [inline src/commands/Syndicates.js]: Syndicates are using the conclave icon

File: src/commands/VoidTrader.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+/**
+ * Displays the currently active Invasions
+ */
+class VoidTrader extends Command {
+  /**
+   * Constructs a callable command
+   * @param {Genesis} bot  The bot object
+   */
+  constructor(bot) {
+    super(bot, 'ondemand.voidtrader', 'trader', 'Display the currentstatus of the void trader');
```
Comment (nspacestd) [inline src/commands/VoidTrader.js]: The command was baro in the previous version, I don't think it should be changed in order not to confuse people

File: src/commands/ConclaveChallenges.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            text: 'Data evaluated by warframe-wordstate-parser, Warframe Community Developers',
+          },
+        };
+        if (categoryInValues) {
+          conclaveChallenges.forEach((challenge) => {
+            if ((challenge.category === category || category === 'all') && !challenge.isRootChallenge()) {
+              embed.fields.push({ name: challenge.mode, value: `${challenge.description} expires in ${challenge.getEndString()}` });
+            }
+          });
+        } else {
```
Comment (nspacestd) [inline src/commands/ConclaveChallenges.js]: `/conclave` without arguments could display all the challenges

File: src/commands/VoidTrader.js (hunk 2 above)
Comment (aliasfalse) [inline src/commands/VoidTrader.js]: sounds good.

File: src/commands/ConclaveChallenges.js (hunk 3 above)
Comment (aliasfalse) [inline src/commands/ConclaveChallenges.js]: my concern on that was that doing it without specifying all would get a bit spammy, and this gives possible values.

File: src/commands/Syndicates.js (hunk 1 above)
Comment (aliasfalse) [inline src/commands/Syndicates.js]: added its own thumbnail

**Pull request merged**